src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/wiremock/PrisonApiMockServer.kt

```diff
@@ -41,8 +41,8 @@ class PrisonApiMockServer : WireMockServer(8979) {
                {
                   "adjudicationNumber": 1524242,
                   "reporterStaffId": 486080,
-                  "bookingId": 1524242,
-                  "offenderNo": "A12345",
+                  "offenderNo": "AA1234A",
+                  "bookingId": 123,
```

Comment:
Jus a slight adjustment to the test data - previously it used the adj number for the booking id